module_3/gift_certificates_system/model/src/main/java/com/epam/esm/entity/Tag.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.esm.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
```

Comment:
Can't see the reason for using this library

Reply:
Thanks, fixed.